ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml

```diff
@@ -32,7 +34,35 @@ periodics:
       - name: JOB_NAME_SAFE
         value: e2e-operator-metrics
       - name: TEST_COMMAND
-        value: make e2e-operator-metrics
+        value: |
+          #!/bin/bash
+
+          git clone https://github.com/openshift/origin.git -b master
+          cd origin
+
+          export GO111MODULE=off
+
+          # install Kubectl required by the test.
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+          mkdir -p /tmp/shared
+          export PATH=$PATH:/tmp/shared
+          mv ./kubectl /tmp/shared/kubectl
+
+          # make and run test
```

Comment:
instead of downloading kubectl, we should also make WHAT=cmd/oc and use that, right?

Reply:
is oc not in the container already?

Reply:
Kubectl is used by `make WHAT=cmd/openshift-tests`. I don't think I can circumvent that. A similar procedure is also used in https://github.com/openshift/origin/blob/master/test/extended/conformance-k8s.sh